mungegithub/mungers/milestone-maintainer.go

```diff
@@ -283,6 +278,32 @@ func durationGreaterThanZero(name string, value time.Duration) error {
 	return nil
 }
 
+// parseMilestoneModes transforms a string containing milestones and
+// their modes to a map:
+//
+//     "v1.8=dev,v1.9=slush" -> map[string][string]{"v1.8": "dev", "v1.9": "slush"}
+func parseMilestoneModes(target string) (map[string]string, error) {
+	if len(target) == 0 {
```

Comment:
It is more robust to verify that `len(result)>0` at the end of the function.

Reply:
Done.